File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.py
def solution(food):
    for i in range(1, len(food)):
        if (food[i] % 2 != 0):
            food[i] -= 1
    
    result = ''
    for i in range(1, len(food)):
        result += (str(i) * int(food[i] / 2))
    
    return result + '0' + "".join(reversed(list(result)))

File: 프로그래머스/1/134240. 푸드 파이트 대회/푸드 파이트 대회.js
function solution(food) {
    for (let i = 1; i < food.length; i++) {
        if (food[i] % 2 !== 0) {
            food[i] -= 1;
        }
    }
    
    let result = '';
    for (let i = 1; i < food.length; i++) {
        result += String(i).repeat(food[i] / 2);
    }
    return result + "0" + result.split("").reverse().join("");
}